tla/consensus/Traceccfraft.tla

```diff
@@ -147,12 +155,20 @@ IsTimeout ==
     /\ logline.msg.state.leadership_state = "Candidate"
     /\ Timeout(logline.msg.state.node_id)
     /\ Range(logline.msg.state.committable_indices) \subseteq CommittableIndices(logline.msg.state.node_id)
+    /\ commitIndex[logline.msg.state.node_id] = logline.msg.state.commit_idx
```

Comment:
This is very repetitive, I would have preferred if it had been factored out.

Reply:
Absolutely, I even started with an `IsState` definition that asserted `log_index`, `membership_state`, `leadership_state`, and` `commit_index`.  I decided to inline the four assertions for as long as not all actions are consistent.